pkg/template/template.go

```diff
@@ -46,7 +46,7 @@ var (
 
 // UserScript encapsulates the data required for creating user script that will be deployed to the instance(s)
 type UserScript struct {
-	InstanceType, VCpus, Memory, Os, Architecture, BucketName, Timeout, BucketRootDir, TargetUtil, CompressedTestSuiteName, TestSuiteName, CustomScript, Region string
+	InstanceType, VCpus, Memory, Os, Architecture, BucketName, Timeout, BucketRootDir, CompressedTestSuiteName, TestSuiteName, CustomScript, Region string
```

Comment:
does this not need the new threshold values in place of TargetUtil?

Reply:
Correct it does not. Before transitioning to using CloudWatch metrics, the Agent code (run on the tested instance) would be responsible for aggregating the data AND determining whether or not it met the TargetUtil.

Now that IQ queries CloudWatch outside of Agent code, the comparison to thresholds will be handled locally (i.e. not on the tested instance).